challenges/solutions_and_support_engineering/README.md

```diff
@@ -0,0 +1,73 @@
+
+# Requirements
+The challenge is set up as a remote troubleshooting session. The candidate and gravitational engineers in similar roles will join a live troubleshooting session via zoom screen sharing. The candidate will be required to solve several faults created in a simulated customer environment. 
+
+Because the hardest troubleshooting we have to do in customer environments is using remote hands without direct access to the system, this challenge is set up to simulate that worst-case experience of walking remote hands by troubleshooting the problems in the environment with only a shared screen.
+
+# Guidance
+## Interview Process
+The interview team and candidate will join a scheduled zoom call for the troubleshooting session, scheduled for 2.5 hours.
+
+- 5m - Intro and Setup
+- 60m - Troubleshooting
+- 10m - Built-in break
+- 60m - Troubleshooting continued
+- 15m - Wrap up, Q&A, etc.
+
+After the session, the interview panel will privately/anonymously submit +1/-2 to the hiring manager, after which the panel will meet to discuss how the session went.
+
+In the case of a positive result, we will connect you to our HR team, collect one-two references, and work out other details.  You can start the reference collection process in parallel if you would like to speed up the process. After reference collection, our ops team will send you an offer. In case of a negative score result, the hiring manager will contact you and share a list of the team's key observations that affected the result.
+
+## Area's of Focus
+We are not looking for any specific application knowledge. The test is designed to troubleshoot generic configuration or system faults on any software deployed to a generic Linux host. While this does not require any deployed application knowledge, some familiarity with collecting logs or investigating unfamiliar software is part of the process.
+
+The faults are present in:
+- kernel / systemd / cgroups
+- filesystem or volumes
+- networking
+- SELinux
+- TLS
+
+We use the following software to represent applications on the host, which are experiencing faults:
+- nginx
+- etcd
+- docker
+
+As this is meant to test the way we normally work, you may use any resource you would normally use when troubleshooting a system; for example, using google as a reference is normal and expected.
+
+## Pitfalls and Gotchas
+Some of the challenges exercise some of the hardest problems we've seen in the wild, so it's common to get stuck on one problem. If you do find you're not making progress on an issue, it's fine to move on and skip a problem and come back to it later.
+
+While this is a simulated environment, the candidate should treat the challenge as troubleshooting a live customer or production system.
+
+It's OK to ask the interview team questions; we'll do our best to provide hints or help steer the investigation if the investigation is going in the wrong direction.
+
+
+## Results
```

Comment:
Seems like this whole section is the same as in the "interview process". Maybe keep it either here or there?